stores/infra.js

```diff
@@ -68,8 +58,6 @@ export const use_infra_store = defineStore("infra", {
       if (isElectron()) {
         await window.electronAPI.run_back(geode_store.PORT)
```

Comment:
Electron should not be used in ogweb?

Reply:
Only if is isElectron returns true, I thought we agreed on that

Reply:
Yes, I missed the isElectron..